apps/extension/src/utils/connectionManager.ts

```
@@ -0,0 +1,382 @@
+import * as vscode from 'vscode';
+import { MCPClientManager, MCPConfig, MCPServerStatus } from './mcpClient';
```

Biome (1.9.4)

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)